scripts/ci/install-go.sh

```diff
@@ -1,25 +1,47 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
-declare -r GO_VERSION=1.16.5
+declare -r GO_VERSION=1.17
+
+# Hashes are from the table at https://golang.org/dl/
```

Comment:
Any easy way to automate pulling the hashes directly? Or too much trouble...

Reply:
I think we'd need to parse the HTML of the page, so probably too much trouble :) having them hard-coded here is (IMO) part of the point anyways, so we don't have to worry about having our request to pull the hashes getting corrupted / intercepted.